bindinfo/handle.go

```diff
@@ -575,6 +578,16 @@ func (c cache) getBindRecord(hash, normdOrigSQL, db string) *BindRecord {
 	return nil
 }
 
+func (c cache) getBindRecord(hash, normdOrigSQL, db string) *BindRecord {
+	bindRecords := c[hash]
+	for _, bindRecord := range bindRecords {
+		if bindRecord.OriginalSQL == normdOrigSQL {
+			return bindRecord
+		}
+	}
+	return nil
+}
+
```

Comment:
Would this logic cause some bad cases?
For example, before updating TiDB, the user has two same bindings in different DBs(DB1 and DB2).
And then the user updates TiDB to this commit and wants to drop the binding in DB1.